cf/test/test_CellMethod.py

```diff
@@ -137,6 +137,12 @@ def test_CellMethod_get_set_delete(self):
         self.assertTrue(cm1.axes == ('time',))
 
 
+    def test_CellMethod_intervals(self):
+        cm, = cf.CellMethod.create('lat: mean (interval: 1 hour)')
```

Comment:
```suggestion
        cm = cf.CellMethod.create('lat: mean (interval: 1 hour)')
```

as the comma has no purpose?